recipes/genera/meta.yaml

```
@@ -0,0 +1,43 @@
+package:
+  name: genera
+  version: "1.4.2"
+
+source:
+  - url: https://github.com/josuebarrera/GenEra/archive/refs/tags/v1.4.2.tar.gz
```

Version could be a variable. Here's canonical boilerplate for this:

```suggestion
{% set version = '1.4.2' %}

package:
   name: genera
   version: {{ version|replace('-', '_') }}

source:
  - url: https://github.com/josuebarrera/GenEra/archive/refs/tags/v{{ version }}.tar.gz
```